horreum-backend/src/test/java/io/hyperfoil/tools/horreum/svc/UtilTest.java

```diff
@@ -9,19 +9,58 @@
 import io.quarkus.test.junit.QuarkusTest;
 import io.quarkus.test.junit.TestProfile;
 import io.quarkus.test.oidc.server.OidcWiremockTestResource;
+import org.jboss.logging.Logger;
+import org.jboss.logmanager.Level;
+import org.jboss.logmanager.LogContext;
+import org.jboss.logmanager.formatters.PatternFormatter;
+import org.jboss.logmanager.handlers.OutputStreamHandler;
 import org.junit.jupiter.api.Test;
+
+import java.io.ByteArrayOutputStream;
 import java.io.IOException;
+import java.io.UnsupportedEncodingException;
 import java.time.Instant;
 
-import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static org.junit.jupiter.api.Assertions.*;
 
 @QuarkusTest
 @QuarkusTestResource(PostgresResource.class)
 @QuarkusTestResource(OidcWiremockTestResource.class)
 @TestProfile(HorreumTestProfile.class)
 public class UtilTest {
 
+    public static class StringHandler extends OutputStreamHandler{
+        ByteArrayOutputStream baos = new ByteArrayOutputStream();
+
+        public StringHandler(){
+            setOutputStream(baos);
+            this.setFormatter(new PatternFormatter("%m"));
+            this.setLevel(Level.ALL);
+            this.setAutoFlush(true);
+            LogContext.getLogContext().getLogger("").addHandler(this);
+        }
+
+        public String getLog(){return baos.toString();}
+
+        @Override
+        public void close() throws SecurityException {
+            LogContext.getLogContext().getLogger("").removeHandler(this);
+            super.close();
+            try {
+                baos.close();
+            } catch (IOException e) { /* meh */}
+        }
+    }
 
+    @Test
+    public void toInstant_nulls() throws UnsupportedEncodingException {
```

Comment:
I am not a fan of mixed CamelCase and snake_case but that is just a minor nitpick :smile: , then up to you.
 
```suggestion
    public void toInstantNulls() throws UnsupportedEncodingException {
```

The same comment for `toInstant_valid` even if outside of this PR.

Reply:
I like `<methodName>_<test_description>` which I think makes it easier to search for the method name as well as easier to read the conditions in the test. 
I find snake_case easier to read for a description but Java methods are usually camelCase, thus the mixed case. There is also the `test<MethodName><TestDescription>` convention but that gets very crowded AndHarderToReadForMyOldEyes. :)

I agree that `src/main` code should adhere to one casing, `cameCase` being the norm for Java, but I think test function names serve to describe what is being tested not to be used as an API.

Reply:
I never used `<methodName>_<test_description>` approach but I see a value in that, maybe we could think about adding some sort of shared/global formatting and/or checks to ensure we all follow the same convention in future improvements

Reply:
https://github.com/Hyperfoil/Horreum/discussions/1538